doc/whats_new/v1.2.rst

```diff
@@ -2,6 +2,21 @@
 
 .. currentmodule:: sklearn
 
+Version 1.2.1
```

Comment:
```suggestion
.. _changes_1_2_1:

Version 1.2.1
```

Reply:
What is the reason for releasing the code with a bug in 1.2.0 since this is only a release? We could directly backport for the next release?

Reply:
Yes let's put that in 1.2

Reply:
The changelog entry has been moved to 1.2.